src/webots/nodes/WbConnector.cpp

```diff
@@ -920,7 +920,7 @@ void WbConnector::hasMoved() {
 // look recursively through Solid and notify each Connector
 void WbConnector::solidHasMoved(WbSolid *solid) {
   // when the simulation is running we don's allow changes
-  if (WbSimulationState::instance()->isRunning() || WbSimulationState::instance()->isFast())
+  if (WbSimulationState::instance()->isFast() || !WbSimulationState::instance()->is3dViewShown())
```

Comment:
The following code doesn't seem to depend on the rendering
```suggestion
  if (WbSimulationState::instance()->isFast()))
```